src/store/annotations/reducer.ts

```diff
@@ -1,19 +1,28 @@
 import { createReducer, combineReducers } from '@reduxjs/toolkit';
 import { AnnotationsState } from './types';
-import { createAnnotationAction, fetchAnnotationsAction, setActiveAnnotationIdAction } from './actions';
+import {
+    createAnnotationAction,
+    deleteAnnotationAction,
+    fetchAnnotationsAction,
+    setActiveAnnotationIdAction,
+} from './actions';
 
 const activeAnnotationId = createReducer<AnnotationsState['activeId']>(null, builder =>
     builder
-        .addCase(setActiveAnnotationIdAction, (state, { payload: annotationId }) => annotationId)
         /* Preview will set the active ID with a an event which will trigger the url change */
-        .addCase(createAnnotationAction.fulfilled, () => null),
+        .addCase(createAnnotationAction.fulfilled, () => null)
+        .addCase(deleteAnnotationAction, (state, { payload: id }) => (state === id ? null : state))
```

Comment:
Should we be setting the active annotation to null if it is deleted? It's kind of the SDK vs app debate we have -- in the create flow, we lean on the application to set the active state once create completes, should we do the same here?

Reply:
I feel like since we are removing a possible active annotation we shouldn't have to rely on a secondary application to clean up the internals of annotations.

Reply:
I see what you're saying. Seems fine to me